spot-electron/package.json

```diff
@@ -0,0 +1,36 @@
+{
+    "author": "Jitsi",
+    "build": {
+        "appId": "org.jitsi.spot",
+        "mac": {
+            "category": "public.app-category.business",
+            "target": "default"
+        },
+        "productName": "Jitsi Spot"
+    },
+    "description": "",
+    "devDependencies": {
+        "babel-eslint": "10.0.1",
+        "electron": "5.0.1",
+        "electron-builder": "20.40.2",
+        "eslint": "5.16.0",
+        "eslint-config-jitsi": "github:jitsi/eslint-config-jitsi#1.0.1",
+        "eslint-plugin-flowtype": "3.8.2",
+        "eslint-plugin-import": "2.17.2",
+        "eslint-plugin-jsdoc": "4.8.4"
+    },
+    "license": "ISC",
+    "main": "main.js",
+    "name": "spot-electron",
+    "productName": "Jitsi Spot",
```

Comment:
Do you need productName here and under build?

Reply:
I thought we do, one for the app and one for the packager, but I removed the one under build and it seems like picking up the one on the top level. Thanks!